JSDemos/Demos/Scheduler/GroupOrientation/description.md

```diff
@@ -1 +1,3 @@
-This example shows the **workWeek** view with different group orientations. Select the *"Vertical Grouping"* tab to&nbsp;see the resource headers organized in&nbsp;a&nbsp;column and the *"Horizontal Grouping"*&nbsp;&mdash; arranged in&nbsp;a&nbsp;row.
\ No newline at end of file
+If resources are assigned to appointments, you can group them by these resources. To do so, use the [groups](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/groups.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups') property to specify an array. Each element of this array is **fieldExpr** of a resource kind. Note that the resource headers' order depends on the resources' order in the [resources](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/') array.
```

Comment:
>If resources are assigned to appointments, you can group them by these resources. 

Please add links to appointments and resources.

>To do so, use the [groups](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/groups.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups') property to specify an array. 

This sentence breaks suddenly. How about changing it a bit as follows: To do this, specify the [groups[]]() array. 

>Each element of this array is **fieldExpr** of a resource kind. 

Let's add a link.
 
>Note that the resource headers' order depends on the resources' order in the [resources]() array.

Once again, I suggest you not to use apostrophes with plurals. How about this: Note that the order of resource headers depends on the order of elements in the **resources[]** array.